### PR TITLE
Remove template engine

### PR DESCRIPTION
**Pull request by rvroon**
This part is not productized by R&D...

### CODE REVIEW

File: docs/self-styling/configuration-page.md [hunk 1]
```diff
@@ -32,22 +31,22 @@ When preparing the git Repository please use the proper directory structure.
...
-  
-The `access` directory contains template sets used to change look and feel of your web applications and mobile clients. Refer to the Access component 
-documentation to learn more about switching the template set per client and contents of a particular template set. The name of the directory is resolved as the 
-name of the particular template set. If the template cannot be found in the directory of the chosen set, the content will be served from the `default` directory.
-The `messages` subdirectory contains message bundles that will be loaded by the Access Engine application. Please note that there is a single messages location
-only for all template sets.  
+#### Access
+The `access` directory contains template sets used to change your web applications and mobile clients' look and feel. Refer to the Access component documentation to learn more about switching the template set per client and the contents of a particular template set. The name of the directory is resolved as the name of the specific template set. If the template cannot be found in the chosen set's directory, the content will be served from the `default` directory.
```
Comment (jashaj) [inline docs/self-styling/configuration-page.md:43]: You can now link to the "Access component documentation" :-)

File: docs/self-styling/configuration-page.md [hunk 2]
```diff
@@ -32,22 +31,22 @@ When preparing the git Repository please use the proper directory structure.
...
+The `messages` subdirectory contains message bundles that the Access Engine application will load. Please note that there is one folder for `messages` that are used by all template sets.  
 
-The `delegation` directory contains resources that allow to customize web experience in DABP. The following resources can be customized:
+#### Delegation
+The `delegation` directory contains resources that allow customizing the web experience in DABP. You can customize the following resources:
```
Comment (jashaj) [inline docs/self-styling/configuration-page.md:47]: What's DABP? Perhaps add a link to the Delegated Adminsitration section?

File: mkdocs.yml [hunk 3]
```diff
@@ -161,7 +161,7 @@ nav:
               - '1.x': products/access/release-notes/1.x.md
       - Self-styling:
           - 'Introduction': self-styling/index.md
-          - 'Onegini Template Styling': self-styling/Tool_Template_Styling.md
+#          - 'Onegini Template Styling': self-styling/Tool_Template_Styling.md
```
Comment (jashaj) [inline mkdocs.yml:164]: Why did you put this in comments?

**Review by jashaj — CHANGES_REQUESTED** [review]

File: mkdocs.yml (hunk 3 above)
Comment (rvroon) [reply mkdocs.yml:164]: yes, it is under discussion if the 'template engine' is part of self-styling, so for now I want to hide it... But completely removing felt like too rigorous...

Comment (jashaj) [reply mkdocs.yml:164]: It's never gone, you can always revert a delete in git.

**Review by jashaj — APPROVED** [review]

**Pull request merged**